communication/mavlink_waypoint_handler.cpp

```diff
@@ -1456,24 +1450,20 @@ static waypoint_local_struct_t waypoint_handler_set_waypoint_from_frame(Mavlink_
 {
     global_position_t waypoint_global;
     waypoint_local_struct_t wpt;
-    local_position_t waypoint_coor;
```

Comment:
For code ease of read, I would also change the upper wpt into something like wpt_local, to ease the understanding in global to local tranformation